### PR TITLE
Fix project metadata in test

### LINKED ISSUE #24106: [x86] Test failed: JIT\CheckProjects\CheckProjects\CheckProjects.cmd
**Job:**
https://ci.dot.net/job/dotnet_coreclr/job/master/view/x86/job/jitstress/job/x86_checked_windows_nt_jitstressregs4/818/

**Failed tests:**
JIT\CheckProjects\CheckProjects\CheckProjects.cmd

**Log:**
```
JIT\CheckProjects\CheckProjects\CheckProjects.cmd [FAIL]
        
        Return code:      1
        Raw output file:      D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\Reports\JIT.CheckProjects\CheckProjects\CheckProjects.output.txt
        Raw output:
        BEGIN EXECUTION
         "D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root\corerun.exe" CheckProjects.exe 
        Scanning projects under D:\j\workspace\x86_checked_w---b7304e59\tests\src\JIT
        D:\j\workspace\x86_checked_w---b7304e59\tests\src\JIT\Directed\rvastatics\RVAOrderingTest.ilproj DebugType-n/a-SuffixNone
        D:\j\workspace\x86_checked_w---b7304e59\tests\src\JIT\Directed\rvastatics\RVAOrderingTest.ilproj Optimize-n/a
        7030 projects, 1 needed fixes, 0 were fixed
        Expected: 100
        Actual: -1
        END EXECUTION - FAILED
        FAILED
        Test Harness Exitcode is : 1
        To run the test:
        > set CORE_ROOT=D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root
        > D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\JIT\CheckProjects\CheckProjects\CheckProjects.cmd
        Expected: True
        Actual:   False
        Stack Trace:
          D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\TestWrappers\JIT.CheckProjects\JIT.CheckProjects.XUnitWrapper.cs(129,0): at JIT_CheckProjects._CheckProjects_CheckProjects_._CheckProjects_CheckProjects_cmd()
        Output:
          
          Return code:      1
          Raw output file:      D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\Reports\JIT.CheckProjects\CheckProjects\CheckProjects.output.txt
          Raw output:
          BEGIN EXECUTION
           "D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root\corerun.exe" CheckProjects.exe 
          Scanning projects under D:\j\workspace\x86_checked_w---b7304e59\tests\src\JIT
          D:\j\workspace\x86_checked_w---b7304e59\tests\src\JIT\Directed\rvastatics\RVAOrderingTest.ilproj DebugType-n/a-SuffixNone
          D:\j\workspace\x86_checked_w---b7304e59\tests\src\JIT\Directed\rvastatics\RVAOrderingTest.ilproj Optimize-n/a
          7030 projects, 1 needed fixes, 0 were fixed
          Expected: 100
          Actual: -1
          END EXECUTION - FAILED
          FAILED
          Test Harness Exitcode is : 1
          To run the test:
          > set CORE_ROOT=D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\Tests\Core_Root
          > D:\j\workspace\x86_checked_w---b7304e59\bin\tests\Windows_NT.x86.Checked\JIT\CheckProjects\CheckProjects\CheckProjects.cmd
```

### PR DESCRIPTION
**Pull request by fadimounir**
I copied the metadata from the ilproj file of a neighboring test (rvastatics1.ilproj).
Fixes #24106

### CODE REVIEW

**Comment by fadimounir** [comment]
@RussKeldorph @tannergooding PTAL

**Comment by RussKeldorph** [comment]
Thanks!

File: tests/src/JIT/Directed/rvastatics/RVAOrderingTest.ilproj [hunk 1]
```diff
@@ -2,17 +2,37 @@
...
     <AssemblyName>$(MSBuildProjectName)</AssemblyName>
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>{95DFC527-4DC1-495E-97D7-E94EE1F7140D}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <CLRTestPriority>1</CLRTestPriority>
+
+    <!-- Test relies on rva_statics support see #2451 -->
+    <DisableProjectBuild>true</DisableProjectBuild>
```
Comment (tannergooding) [inline tests/src/JIT/Directed/rvastatics/RVAOrderingTest.ilproj]: This disables the project from building, and I believe disables the test from running.

Is this desired/applicable considering the test was just added?

Comment (tannergooding) [reply tests/src/JIT/Directed/rvastatics/RVAOrderingTest.ilproj]: Also, it looks like some bug fixes have gone in related to https://github.com/dotnet/coreclr/issues/2451; is the issue still applicable to the other disabled tests?

Comment (fadimounir) [reply tests/src/JIT/Directed/rvastatics/RVAOrderingTest.ilproj]: You are right. The test passes when i run it locally (unlike the neighboring ones which fail for me). I'll remove this.

**Pull request merged**